### PR TITLE
Update go to 1.23

### PR DESCRIPTION
**Pull request by Ex4amp1e**
Issue: https://github.com/networkservicemesh/deployments-k8s/issues/12294

### CODE REVIEW

File: internal/vppinit/vppinit.go [hunk 1]
```diff
@@ -140,11 +143,11 @@ func getKernelVer() ([2]int, error) {
 	if err != nil {
 		return [2]int{}, err
 	}
-	min, err := strconv.Atoi(ver[1])
+	minValue, err := strconv.Atoi(ver[1])
```
Comment (denis-tingaikin) [inline internal/vppinit/vppinit.go:146]:
```suggestion
	minor, err := strconv.Atoi(ver[1])
```

File: internal/vppinit/vppinit.go [hunk 2]
```diff
@@ -140,11 +143,11 @@ func getKernelVer() ([2]int, error) {
...
 	if err != nil {
 		return [2]int{}, err
 	}
-	return [2]int{maj, min}, nil
+	return [2]int{maj, minValue}, nil
```
Comment (denis-tingaikin) [inline internal/vppinit/vppinit.go:150]:
```suggestion
	return [2]int{major, minor}, nil
```

File: internal/vppinit/vppinit.go (hunk 1 above)
Comment (Ex4amp1e) [reply internal/vppinit/vppinit.go]: Fixed

File: internal/vppinit/vppinit.go (hunk 2 above)
Comment (Ex4amp1e) [reply internal/vppinit/vppinit.go]: Fixed

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**